src/lib/components/primitives/form/form.svelte

```diff
@@ -29,38 +29,47 @@
     // Iterate over all children of the form, and pluck out the the inputs
```

Comment:
This is the form component that gives us the capability to do client-side validation. So, if AM provides metadata on callbacks informing us that they are required or have min/max values ... whatever, we can implement it using the native DOM validation. Whatever slips through will be caught be AM and that validation will be handled separately.

Reply:
Think this makes sense